### ISSUE TITLE
Add ISO 3166 to ISO 11784 and ISO 11785 animal reading in the 125 kHz RFID app

### ISSUE THREAD

**Issue by Jolexer** [flipperdevices/flipperzero-firmware#4044]
### Describe the enhancement you're suggesting.

When reading an animal as described in [https://docs.flipper.net/rfid/animal-microchips](https://docs.flipper.net/rfid/animal-microchips), the country code it displays is non-descript. These country codes are defined in [ISO 3166](https://docs.flipper.net/rfid/animal-microchips) (wikipedia). Please display the country number along with the english country name in the full information display. In the abreviated info screen, it may be better to show the ISO 3166-A-2 country abbreviation also defined in that wikipedia page, instead of the full english name, to better fit the UI.

Below is a are current screenshots and their proposed versions:
![Screenshot 2024-12-23 at 16 52 11](https://github.com/user-attachments/assets/796444ea-f74a-43df-b777-d00550ebd1fa)
![Screenshot 2024-12-23 at 16 52 11 edit](https://github.com/user-attachments/assets/d87d11c0-526a-4738-853d-ce9973f92b1c)
![Screenshot 2024-12-23 at 17 28 11](https://github.com/user-attachments/assets/91eefeac-115c-4b39-aa32-4e4f24ac30ab)
![Screenshot 2024-12-23 at 17 24 21 edit](https://github.com/user-attachments/assets/c2e0ba95-16a5-43b9-8aa2-c4f986d4e76f)

### Anything else?

I am really grateful that the flipper can read dog tags. Many people don't return collarless dogs because they don't want to bring them to a vet. Having given this feature to the flipper wil help many dogs find their owners.

**Comment by equipter** [comment]
worth noting the first 3 digits "country code" are not always a country code. 

https://mypethq.io/a-guide-to-pet-microchips/

so if this is added it might be worth it to replace country code with manufacturer code if the 3digit code found is not a country code.

**Comment by Jolexer** [comment]
> worth noting the first 3 digits "country code" are not always a country code.

That is true, but they all start with 9xx. You could either put the mfg names in the list as well, or just display "invalid" instead of a country (which should be done for invalid codes anyway).

**Comment by skotopes** [comment]
I'll put it into backlog, but no promises.

**Comment by Jolexer** [comment]
Thank you. Please use this thread for any updates.
